overlays/NetworkManager/rootfs/etc/NetworkManager/system-connections/ww4-managed.ww

```diff
@@ -70,8 +71,11 @@ dns-search={{ join ";" (without (regexSplit "[ ;]+" $netdev.Tags.DNSSEARCH -1) "
 {{- end }}
 
 [ipv6]
-addr-gen-mode=stable-privacy
+{{- if $netdev.Tags.master }}
+method=disabled
+{{- else }}
 method=ignore
+addr-gen-mode=stable-privacy
```

Comment:
Why explicitly set this?  I often set my nodes to `eui64`

Reply:
It was part of the original (September 2022) commit at d57128dbd76527c119f615575bd3fae4e9423532. I'm 100% in support of changing it / further parameterizing it; but I think that should be done separately.